e2e/cypress/integration/enterprise/saml/okta_login_spec.js

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
Can we add this comment so that the required setup steps can be done as mentioned in the reference doc:

`Setup required: Follow the instructions mentioned in the mattermost/platform-private/config/saml-okta-setup.txt file`